tests/integration/conftest.py

```diff
@@ -330,8 +330,17 @@ def is_mysql_ready():
         build_and_deploy_wordpress(),
         wait_mysql_pod_ready(),
         ops_test.model.deploy("charmed-osm-mariadb-k8s", application_name="mariadb"),
-        ops_test.model.deploy(
-            "nginx-ingress-integrator", "ingress", trust=True, channel="edge", series="focal"
+        # temporary fix for the CharmHub problem
+        ops_test.juju(
+            "deploy",
```

Comment:
This and the `.deploy` call look identical?

Reply:
Ah, do you think that should be added to the comment as well? Just so we know what to look for in the future